custom-plugin-decorators/sorting/README.md

```diff
@@ -0,0 +1,94 @@
+# A suite of sorting decorators
+
+Authors:
+- [`@lornajane`](https://github.com/lornajane), Lorna Mitchell (Redocly)
+ 
+## What this does and why
+
+There are lots of reasons that you'd want to alter the order of the items in your API description, such as putting required fields first, or just ordering things alphabetically (or logically!) to make things consistent and easier to find.
+When Redocly only made API documentation tools, the sorting changes were made as part of the docs build.
+But now we make more complex tools, and many API pipelines have more than just docs in them too - so these operations are more commonly done with a decorator to transform the API description; then it can be used with any tools.
+
+Redocly CLI has some rules about sorting, that these decorators can help to satisfy:
+
+- [`tags-alphabetical`](https://redocly.com/docs/cli/rules/tags-alphabetical/) rule to error if the `tags` section isn't in alphabetical order by `name`.
+- [`required-props-first`](https://redocly.com/docs/cli/rules/tags-alphabetical/) rule to error if an object's properties don't put the required fields first.
```

Comment:
This links to tags-alphabetical

Reply:
I don't think we have such a rule for `required-props-first` (or I can't find it).

Reply:
Huh! I guess I should write it as well in that case! It was on my notes, but maybe it was a "todo"?

Reply:
Added a `property-sort` rule